### DISCUSSION TITLE
Add or remove ComboBoxItems to exists ComboBox

### DISCUSSION THREAD

**Discussion by romanjetich** [UI5/webcomponents-react#3269] [Q&A] [answered]
Hi,

i'm new in react and ui5. I look for the possibility to add or remove the items to an already created combobox.
And how do i determine the combobox was added to a dom?

Thanks in andvance

**Comment by Lukas742** [comment]
Hi @romanjetich 

to add or remove components from the DOM you can use [conditional rendering](https://www.w3schools.com/react/react_conditional_rendering.asp). (basic codeSandbox example can be found [here](https://codesandbox.io/s/relaxed-engelbart-ejjs7t?file=/src/App.js))

> And how do i determine the combobox was added to a dom?

Could you elaborate on what you mean with this?

**Comment by romanjetich** [comment]
Thank you for reply. Conditional Rendering is not this what i need. For example I have two combos - cb1 and cb2. And cb1 controls which data will be loaded into the cb2 via request to the server.

**Comment by Lukas742** [comment, marked as answer]
So if I understood correctly, you want to fill the second combobox depending on the response of the server, right?
E.g.:
In cb1 you select "countries" then cb2 should contain all countries from the response of the server.

If that's the case you can just map the response to the `ComboBoxItem`s. See [here](https://reactjs.org/docs/lists-and-keys.html#rendering-multiple-components)

If not please elaborate more on what exactly you want to achieve.
